clearwater-diags-monitor/usr/share/clearwater/bin/clearwater_diags_monitor

```diff
@@ -259,13 +289,15 @@ get_network_info()
 {
   copy_to_dump "/etc/hosts"
 
-  ifconfig -a > $CURRENT_DUMP_DIR/ifconfig.txt
+  # For some reason, redirecting ifconfig to file doesn't work on CentOS, so we
+  # pipe through tee instead.  We do the same below.
```

Comment:
I really don't know - I spent ages looking at this last night.  My only guess is that it's something about the term type of stdout (as ifconfig perceives it).